packages/compartment-mapper/src/link.js

```diff
@@ -374,7 +376,7 @@ export const link = (
     compartmentDescriptor.modules = modules;
 
     const parse = mapParsers(
-      languageForExtension,
+      { ...languageForExtension, ...extraParsers },
```

Comment:
Are we sure we want this to `override`? Could we install `fallback`? I could buy both.

Reply:
_fallback_ would work for my use case, but _override_ may be more useful in the general case (i.,e. some other use-case that I can't be bothered to define precisely).  I think we can go with _fallback_ for now.